src/containers/MultidisciplinarySubject/MultidisciplinarySubjectArticlePage.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import Spinner from '@ndla/ui/lib/Spinner';
+import { useGraphQuery } from '../../util/runQueries';
+import { topicQueryWithPathTopics } from '../../queries';
+import { getUrnIdsFromProps } from '../../routeHelpers';
+import MultidisciplinarySubjectArticle from './components/MultidisciplinarySubjectArticle';
+
+const MultidisciplinarySubjectArticlePage = ({ match, locale }) => {
+  const { topicId } = getUrnIdsFromProps({ match });
```

Comment:
Her mangler du en parameteter fra min pr.
```suggestion
  const { subjectId, topicId } = getUrnIdsFromProps({ match });
```